torch/distributed/_spmd/gm_transformation.py

```diff
@@ -17,6 +17,7 @@
 )
 from torch.distributed._spmd.graph_utils import dump_graphs_to_files, OP
 from torch.distributed._spmd.iter_graph_module import IterGraphModule
+from torch.distributed._spmd.partial_lower import partial_lower
```

Comment:
maybe worth a comment that this is expected to be manually copied over? (to avoid confusion)

Reply:
I added the comment in `partial_lower.py`